mautic/mautic
# Fix Email sent, DNC and UTM report filters

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ Y ]
| New feature/enhancement? (use the a.x branch)      | [ N ]
| Deprecations?                          | [ N ]
| BC breaks? (use the c.x branch)        | [ N ]
| Automated tests included?              | [ Y ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This PR restores filters that were accidentally removed from Email sent, DNC and UTM reports in #11346.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create new Report using the Emails Sent Datasource
3. Go to the Data Tab, click on the Add Filer button
4. Check that the following filters appear in the list: Bounced, Campaign, Campaign ID, Category ID, Category name, Clicks, Clicked, Email
5. Switch datasource to Do Not Contact
6. Check that the following filters appear in the list: DNC Channel, DNC Channel ID, DNC, Comment
7. Switch datasource to UTM Codes
8. Check that the following filters appear in the list: UTM Campaign, UTM Medium, etc.
